.changeset/hip-wasps-accept.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': patch
+'@itwin/itwinui-css': patch
+---
+
+Fixed an issue where `Table` was showing a dummy scrollbar track inside the table header in Chrome 121.
```

Comment:
Should we mention Chrome 121 *+ Windows*?

```suggestion
Fixed an issue where `Table` was showing a dummy vertical scrollbar track inside the table header in Chrome 121 + Windows.
```

Reply:
Actually, this also happened to me on Edge 121 + Windows.

Reply:
Edge is just Chromium under the hood, so not necessary, but mentioning Windows makes sense 👍  [`13f29d6`](https://github.com/iTwin/iTwinUI/pull/1816/commits/13f29d6fcbbd9407f06aba6ef3f28122f51c94fe)